python/cudnn/sdpa/fwd/api_dsl.py

```
@@ -1566,6 +1573,23 @@ def _scalar(t, default=1.0):
+            return self._execute_fp8_thd(
+                q_tensor,
+                k_tensor,
+                v_tensor,
+                o_tensor,
+                lse_tensor,
+                scale_softmax_log2,
+                o_scale_fused,
+                sinks,
+                seq_kv_lens,
+                seq_q_lens,
+                amax_s,
```

Ruff (0.16.1)

[error] Undefined name `amax_s`

(F821)